homeassistant/components/homematic/sensor.py

```diff
@@ -111,6 +245,13 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
 class HMSensor(HMDevice, SensorEntity):
     """Representation of a HomeMatic sensor."""
 
+    def __init__(self, conf):
+        """Initialize HMSensor."""
+        super().__init__(conf)
+        self.entity_description = SENSOR_DESCRIPTIONS.get(self._state)
+        if self.entity_description is None:
```

Comment:
Let's not make a fallback. At one point we have to fix that to having for all sensors the correct description object. So if somethings pop-up, we can fix it

Reply:
Without the default description it will crash, and then the sensor does not work. Maybe log an error and still do the copy? 
![grafik](https://user-images.githubusercontent.com/12832850/139723836-855d3dcd-3464-4e9e-835a-a0a2451fbd1d.png)
I would open another PR to maintain those missing sensor descriptions. Or what do you think?

Reply:
Okay but let's do that on the setup platform and update the base HMDevice `__init__(conf: Dict[str, str], description: EntityDescription | None = None)` So we don't need overload the init on each platform (future)

Reply:
Done. Thanks for review 👍